### PR TITLE
[#32] - AM/AG - Add waffle.io and gitter badges to readme

### PR DESCRIPTION
**Pull request by alxmrtn**

### CODE REVIEW

**Comment by ghost** [comment]
Hi there, I'm whereat-gobot! Where's Swarmzo???

**Comment by aguestuser** [comment]
Hey so @alxmrtn : can we close this PR since it was sort of an experiment in breaking the build? Does it have anything important that pull request #35 depends on?

**Comment by aguestuser** [comment]
@alxmrtn : whoops! just saw that you unbroke the build! removing the circle integration for now (realize what you mean about the confusion!) other than that, this PR should be good to merge, yes?

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+[![Stories in Ready](https://badge.waffle.io/whereat/wiki.svg?label=ready&title=open tickets)](http://waffle.io/whereat/whereat-location-server)
+[![Gitter](https://badges.gitter.im/whereat/whereat-location-server.svg)](https://gitter.im/whereat/whereat-location-server?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=body_badge)
```
Comment (aguestuser) [inline README.md:2]: Actually, just realizing that this is wrong: there is no `gitter.im/whereat/whereat-location-server` chat room. Only the main room at `gitter.im/whereat/contrib` gonna close and not merge this and circle back to adding bages to _all_ repos in one big batch set of PR's. @alxmrtn : cool?

**Pull request closed without merging**